ui/tests/integration/components/info-table-row-test.js

```diff
@@ -114,7 +114,7 @@ module('Integration | Component | InfoTableRow', function (hooks) {
       @type={{this.type}}
     />`);
 
-    assert.dom('[data-test-item="array"]').hasText('valueArray', 'Confirm link with item value exist');
+    assert.dom('[data-test-item="valueArray"]').hasText('valueArray', 'Confirm link with item value exist');
```

Comment:
because I changed this to be dynamic and take on the role name instead of a hardcoded "array"